### ISSUE TITLE
Record email notifications in the Activity Log

### ISSUE THREAD

**Issue by ahamelers** [CDL-Dryad/dryad-product-roadmap#2153]
Separated from #2083, suggested by @jleighherzog 

> Additionally, would it be possible to capture all auto-notifications sent in the Activity Log? It would be helpful to view all actions in the history for each dataset.

**Comment by jleighherzog** [comment]
@ahamelers : Before this is completed, do you need more specifics?

**Comment by ahamelers** [comment]
@jleighherzog I've put in code to add a short message each time an email is sent to the author or the journal, like

> Journal sent status change notification

or

> Author sent in progress reminder notification

Do you think this will be sufficient? Phrasing okay? I thought we might need to ask questions about when to add the note to the curation activities, but it ended up being pretty easy to add it whenever there is an email sent out that's associated with a specific submission. 😄

**Comment by ryscher** [comment]
To avoid possible confusion, I would reword as "Status change notification was sent to journal"

**Comment by jleighherzog** [comment]
@ahamelers : This might need a call to discuss too. "Status change notification was sent to journal" doesn't equal "Journal sent status change notification" -- those seem to be different communications to different parties. Let's include this in our call re: #2083?

**Comment by ahamelers** [comment]
@jleighherzog That's exactly the confusion Ryan was talking about—this ticket is only about notifications sent by the system to journals and authors, so we have used the wording "Status change notification was sent to [recipient]" as Ryan suggested
